docs/user-guide/helics_co-sim_sequence.md

```diff
@@ -0,0 +1,84 @@
+ 		"type":"double", 
+ 		"required":true 
+	}],
+  }
+  ```
+
+3. **Launch co-simulation/Initialize federates** - This will create the federates as entities recognized by the broker, set-up the communication channels for their messages to be passed, pass some initial messages and execute some preliminary code as preparation for the beginning of the co-simulation proper. The later is particularly important if the various federates need to reach a self-consistent state as an initial condition of the system. 
+
+   In our example, it may be important that when the co-simulation proper begins (t = 0) that the loads and resulting voltages that are solved for across both the transmission and distribution systems are in agreement. Or, depending on the specific purpose of the co-simulation, it may be acceptable for the first few powerflows to be less precise as the federates step through time towards a more consistent state across the federation.
+
+4. **Request Time** - Once the initializing phase is complete, the co-simulation proper begins. Every federate, based on the system it is modeling and what information it needs to that simulation, will request a simulated time from the broker. This time request indicates the point in simulated time at which the federate knows it will have to do run so it can simulate some portion of the model or behavior in the system. For example, there may be a federate simulating a building and based on the dynamics of the system, it know the indoor temperature will not appreciably change over the next five minutes. More detailed discussion of time management is found in the [section on timing](./timing.md).
+
+    In the case of running our integrated transmission and distribution powerflow, both federates only need to update when the other provides a new value. Effectively, the time request will determine the temporal resolution of the simulation. If loadshapes or historical data is being used by the distribution system to determine individual load co-efficients, the resolution of that data would define a lower bound for the temporal resolution. For this example, let's assume each federate only needs to update every five minutes.
+
+5. **Time Grant** - Based on the time requests and grants from all the connected federates, a core will determine the next time it can grant to a federate to guarantee none of the federates will be asked to simulate a point back in time. That is, if the time coordinator is doing its job correctly, every federate will receive a time that is the same as or larger than the last time it was granted. HELICS does support a configuration and some other situations that allows a federate to break this rule but this is a very special situation and would require all the federates to support this jumping back in time, or accept non-causality and some randomness in execution. This and other aspects of granting time are discussed in more detail in [the section on timing](./timing.md).
```

Comment:
I modified this section quite a bit.  It didn't really convey one of the key things about HELICS in that there is no central timekeeper.  Everything is distributed, each federate has there own timeCoordinator which makes the grant decision based on the connections.  This section previously implied a centralized time keeping algorithm  which was fundamentally false.    The brokers really have very little to do with the time.